client/src/API/index.js

```diff
@@ -0,0 +1,38 @@
+import axios from 'axios';
+import axiosConfig from '../config/axiosConfig';
+
+const createPost = async (body) => {
+  let response;
+  try {
+    response = await axios.post('posts', body, axiosConfig);
+    if (!response.data.success) {
+      throw new Error(response.data.message);
+    }
+  } catch (err) {
+    console.error(err);
```

Comment:
Remove consoles and show appropriate error messages on screen

Reply:
Placeholders for now. Will replace later